api/python/cellxgene_census/src/cellxgene_census/experimental/pp/_highly_variable_genes.py

```diff
@@ -206,6 +206,7 @@ def _highly_variable_genes_seurat_v3(
     median_ranked = np.ma.median(ma_ranked, axis=0).filled(np.nan)  # type: ignore[no-untyped-call]
 
     var_df = var_df.assign(
+        gene_name=var_df.index,
```

Comment:
I don't quite follow this code.  It appears to assign the soma_joinid of each var column to the `gene_name`.  While this puts something in the column, it doesn't really follow the spirit of the ScanPy implementation or our CxG schema.

In the CxG schema, the var index contains the ENSEMBL ID of the gene (aka `feature_id` in the Census var dataframe). This is similar to most (*not* enforced) AnnData usage.

For ease of use, I argue that we should assign `gene_name` to be the `feature_id` value, not the `soma_joinid`.  You can look up feature_id from soma_joinid (and vice versa), but the feature_id is more likely what the user expects.

This is an API design issue for @pablo-gar and @hthomas-czi